options/options.go

```diff
@@ -768,20 +775,32 @@ func cloneInto(dest proto.Message, src proto.Message, res linker.Resolver) error
 	if dest.ProtoReflect().Descriptor() == src.ProtoReflect().Descriptor() {
 		proto.Reset(dest)
 		proto.Merge(dest, src)
-		return proto.CheckInitialized(dest)
+		return nil
```

Comment:
Three changes in this method were made to avoid redundant checks about missing required fields. We already report that from the call to `validateRecursive`, so we don't need to report more errors due to this serialization work failing.